test/dump/expr-br.txt

```diff
@@ -2,7 +2,7 @@
 ;;; FLAGS: -v
 (module
   (func (result i32)
-    block i32
+    block (result i32)
```

Comment:
The purpose is to allow for future language extensions where the block signature is more complex (potentially with params and results).